doc/source/whatsnew/v0.17.0.txt

```diff
@@ -101,56 +104,55 @@ Old                     New
 =====================   =============
 
 - Coercing types with ``DataFrame.convert_objects`` is now implemented using the
-keyword argument ``coerce=True``.  Previously types were coerced by setting a
-keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
+  keyword argument ``coerce=True``.  Previously types were coerced by setting a
+  keyword argument to ``'coerce'`` instead of ``True``, as in ``convert_dates='coerce'``.
 
-  .. ipython:: python
+.. ipython:: python
```

Comment:
it is a bit difficult to see in this diff, but I think it was correct. You want the code-block to 'belong' to the list item, so the `..` has to start at the same indentation level as the other text